File: cl/cl6/uri1002.cpp
#include <iostream>

const double PI = 3.14159;
double radius;

bool read_data() {
  bool flag = true;

  std::cin >> radius;
  if (radius == 0) flag = false;

  return flag;
}

void process_data() {
  std::cout << "A: " << PI * radius * radius << '\n';
}

int main() {
  bool flag = true;

  flag = read_data();
  while (flag) {
    process_data();
    flag = read_data();
  }

  return 0;
}

File: data/pc/cl6/uri1002.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class uri1002 {
  static InputStreamReader ir = new InputStreamReader(System.in);
  static BufferedReader in = new BufferedReader(ir);
  static final double PI = 3.14159;
  static double radius;
  static double result;

  static boolean readData() throws IOException {
    boolean flag = true;

    try {
      radius = Double.parseDouble(in.readLine());
    }
    catch (Exception NumberFormatException) {
      radius = 0;
    }

    if (radius == 0) {
      flag = false;
    }

    return flag;
  }

  static void processData() {
    result = PI * radius * radius;
  }

  static void writeData() {
    System.out.printf("A=%.4f\n", result);
  }

  public static final void main(String[] args) throws IOException {
    boolean flag = true;

    flag = readData();
    while (flag) {
      processData();
      writeData();
      flag = readData();
    }
  }
}
